port/ztpf/omrvmem.c

```diff
@@ -1445,6 +1445,14 @@ getMemoryInRangeForDefaultPages(struct OMRPortLibrary *portLibrary,
 					PPG_vmem_pageSize[0], category);
 		}
 
+/**
+ * Disabling address range checking for z/TPF.  Current z/TPF support for mmap
+ * is non-standard.  Due to the non standard behaviour the address range
+ * checking may cause an infinite loop in some scenarios where the mapping
+ * fails.  The address range feature maybe re-enabled in the future once
+ * z/TPF mmap support is updated.
```

Comment:
Removed double space, added missing punctuation and minor rewording.
```suggestion
 * Disable address range search for z/TPF. Current z/TPF support for mmap
 * is non-standard. Due to the non standard behaviour, the address range
 * search may cause an infinite loop in some scenarios where mmap fails.
 * The address range search maybe re-enabled in the future once z/TPF
 * mmap support is updated.
```